lib/payday/invoiceable.rb

```diff
@@ -46,6 +46,7 @@ def shipping
 
   # Calculates the total for this invoice.
   def total
+    return calculated_total if defined?(calculated_total) && calculated_total
```

Comment:
I guess we'll have some issue with merging the previous branch right?

Reply:
@alonhartal why? in case of tax?